library/request_queue.cpp

```diff
@@ -104,6 +106,11 @@ dnet_io_req *dnet_request_queue::pop_request(dnet_work_io *wio, const char *thre
 	if (!expired)
 		return r;
 
+	// Note that marked with `weak` attribute and not defined in client bindings,
```

Comment:
```cpp
// Note: this function is marked with `weak` attribute and is not defined in client bindings,
// so this code should not be executed from client side otherwise it will lead to segfault.
```